### PR TITLE
MAUI-3387 : ListView - Maintain the scroll position while updating ItemsSource at runtime

### PR DESCRIPTION
**Pull request by RiyashameedM**
Modified Maintain the scroll position while updating ItemsSource at runtime topic.

Got content review
https://syncfusion.atlassian.net/browse/DOC-4946

### CODE REVIEW

File: MAUI/ListView/scrolling.md [hunk 1]
```diff
@@ -183,52 +183,15 @@ public partial class MainPage : ContentPage
...
-
-For `Horizontal` orientation, use the `ScrollX` value of `ListViewScrollView`.
-
-By using [Reflection](https://docs.microsoft.com/en-us/dotnet/api/system.reflection?redirectedfrom=MSDN&view=net-5.0), get the value of `ScrollOwner` from `VisualContainer` and use it.
+The `SfListView` have scrolled to top automatically when changing the `ItemsSource` at runtime. However, you can maintain the same scrolled position by using `SfListView.CanMaintainScrollPosition` property.If you set `SfListView.CanMaintainScrollPosition` to true then on changing ItemsSource, the newly added ItemsSource will be loaded with the previous ItemsSource’s ScrollOffset.
```
Comment (florenceliliana) [inline MAUI/ListView/scrolling.md:186]: The `SfListView` has scrolled to the top automatically when changing the `ItemsSource` at runtime. However, you can maintain the same scrolled position using the `SfListView.CanMaintainScrollPosition` property. If you set `SfListView.CanMaintainScrollPosition` to true, then on changing ItemsSource, the newly added ItemsSource will be loaded with the previous ItemsSource’s ScrollOffset.

Comment (RiyashameedM) [reply MAUI/ListView/scrolling.md]: resolved

File: MAUI/ListView/getting-started.md [hunk 2]
```diff
@@ -49,14 +49,14 @@ public partial class MainPage : ContentPage
 
 ## Register the handler
 
-To use this control inside an application, you must initialize the `SfListView` handler.
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/ListView/getting-started.md]: resolved

**Review by PiruthivirajM — APPROVED** [review]
Fine

**Pull request merged**